extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -602,41 +606,45 @@ public static void deleteTable(File path) {
      * key} order) location found will be used to infer schema.
      *
      * <p>
-     * Delegates to one of {@link #readSingleFileTable(File, ParquetInstructions)},
+     * Delegates to one of {@link #readSingleFileTable(URI, ParquetInstructions)},
      * {@link #readPartitionedTableWithMetadata(File, ParquetInstructions)},
      * {@link #readFlatPartitionedTable(File, ParquetInstructions)}, or
      * {@link #readKeyValuePartitionedTable(File, ParquetInstructions)}.
      *
-     * @param source The source file or directory
+     * @param source The URI for source file or directory
      * @param instructions Instructions for reading
      * @return A {@link Table}
      */
     private static Table readTableInternal(
-            @NotNull final File source,
+            @NotNull final URI source,
             @NotNull final ParquetInstructions instructions) {
```

Comment:
I think changing this signature is confusing - I _could_ see it changing to `Path`, but given the mostly existing impl, I'm confused. I think it's better to have a new method that delegates to this File/Path based impl instead.